packages/rs-drive-abci/src/execution/check_tx/v0/mod.rs

```diff
@@ -103,13 +106,22 @@ where
             block_info,
         };
 
-        let execution_event = process_state_transition(&platform_ref, state_transition, None)?;
-
         let platform_version = platform_ref.state.current_platform_version()?;
 
+        let execution_event = check_tx_state_transition_to_execution_event(
```

Comment:
Just `check_state_transition`?

Reply:
No, because it's a state_transition_to_execution_event, but for check_tx

Reply:
Agreed to rename to make it more understandable